kuyruk_sentry.py

```diff
@@ -13,9 +13,6 @@
 class Sentry(object):
     def __init__(self, kuyruk):
         kuyruk.extensions["sentry"] = self
-        scope = sentry_sdk.Scope()
-        client = sentry_sdk.Client(kuyruk.config.SENTRY_DSN)
-        self.hub = sentry_sdk.Hub(client, scope)
```

Comment:
It seems like one of these messes with existing Sentry configuration in `>=2.0.0`.

We configure Sentry with `LoggingIntegration(level=logging.INFO, event_level=logging.CRITICAL)`, then after initializing `kuyruk_sentry.Sentry` it reverts back to `LoggingIntegration(level=logging.INFO, event_level=logging.ERROR)` which is the default. The same probably applies with other default integrations.